src/components/Show.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from "next/link";
+import ShowDetail from "./ShowDetail";
+
+type Row = {
+  rows: Rows[];
+};
+
+type Rows = {
```

Comment:
複数形じゃなくて単数形のほうが良いですね。1個分のデータなので

```suggestion
type Row = {
```

Reply:
https://github.com/tatata05/next-app/pull/7/commits/e812ee0ee0e31c4519675f3c098f560c27af0a4b
変更しました！
Rowだと他の型定義名と競合するので、RowTypeにしました。